### PR TITLE
Add new patches to custom loadout

### PR DESCRIPTION
**Pull request by DynastyIV4**
# About the pull request

Added the Solar Devils patch to custom loadout in character preferences

# Explain why it's good for the game

It's just a little and fun change. I think it can be good for character developpement, like some marines that were before in those units.
Also in the US military, you are autorised to wear the patches of your old units, as long as your current one is visible, so it does not break immersion. 

It's my first PR, please tell me if anything is wrong :)

# Changelog

:cl:
code: Added the Solar Devils patche to the clothing accessories for marines

/:cl:

### CODE REVIEW

**Review by harryob — APPROVED** [review]
code is all good

**Comment by c21b4** [comment]
> code is all good

So, how does it work now? Do I have to do anything?

**Comment by harryob** [comment]
> > code is all good
> 
> So, how does it work now? Do I have to do anything?

nothing at all, prs just have a 1 day wait period from being opened till being merged

File: code/modules/client/preferences_gear.dm [hunk 1]
```diff
@@ -975,6 +975,14 @@ GLOBAL_LIST_EMPTY(gear_datums_by_name)
 	display_name = "Falling Falcons shoulder patch"
 	path = /obj/item/clothing/accessory/patch/falcon
 
+/datum/gear/misc/patch_uscm/devils
+	display_name = "USCM Solar Devils patch "
```
Comment (vero5123) [inline code/modules/client/preferences_gear.dm:979]:
```suggestion
	display_name = "USCM Solar Devils patch"
```

File: code/modules/client/preferences_gear.dm [hunk 2]
```diff
@@ -975,6 +975,14 @@ GLOBAL_LIST_EMPTY(gear_datums_by_name)
...
+	path = /obj/item/clothing/accessory/patch/devils
+
+/datum/gear/misc/patch_uscm/forecon
+	display_name = "USCM Force Reconnaissance patch"
+	path = /obj/item/clothing/accessory/patch/forecon
```
Comment (harryob) [inline code/modules/client/preferences_gear.dm:984]: forecon is a bit too special to have marines transferring to the almayer, probably wanna get rid of that one

**Review by harryob — CHANGES_REQUESTED** [review]

File: code/modules/client/preferences_gear.dm (hunk 2 above)
Comment (DynastyIV4) [reply code/modules/client/preferences_gear.dm:984]: Sorry for the late reponse done

Comment (DynastyIV4) [reply code/modules/client/preferences_gear.dm:984]: Nevermind I did an oopsie, resolved agan

File: code/modules/client/preferences_gear.dm [hunk 3]
```diff
@@ -975,6 +975,14 @@ GLOBAL_LIST_EMPTY(gear_datums_by_name)
...
 	path = /obj/item/clothing/accessory/patch/falcon
 
+/datum/gear/misc/patch_uscm/devils
+	display_name = "USCM Solar Devils patch"
+	path = /obj/item/clothing/accessory/patch/devils
+
+/datum/gear/misc/patch_uscm/forecon
+	display_name = "USCM Force Reconnaissance patch"
+	path = /obj/item/clothing/accessory/patch/forecon
+
```
Comment (harryob) [inline code/modules/client/preferences_gear.dm:985]:
```suggestion
```

**Review by harryob — CHANGES_REQUESTED** [review]

**Review by harryob — APPROVED** [review]

**Pull request merged**